src/libraries/System.DirectoryServices/src/System/DirectoryServices/ActiveDirectory/DirectoryServerCollection.cs

```diff
@@ -14,7 +15,7 @@ public class DirectoryServerCollection : CollectionBase
         internal readonly DirectoryContext context;
         internal bool initialized;
         internal readonly Hashtable? changeList;
-        private readonly ArrayList _copyList = new ArrayList();
+        private readonly List<object> _copyList = new List<object>();
```

Comment:
```suggestion
        private readonly List<object> _copyList = new();
```

Do we like target type new in dotnet/runtime? I haven't seen it being used much as yet.

Reply:
I see a few hundred instances of it in newer code, so I guess we like it.

Reply:
We're fine with it in cases like this where the type is still named as part of the operation. We're less happy with it when the code is just like:
```C#
somelocalorfield = new();
```
as that makes it much harder to understand at a glance what's happening.